### PR TITLE
fix(TBD-13622): add tez libraries

### PR DESCRIPTION
**Pull request by sanulich**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

File: main/plugins/org.talend.hadoop.distribution.emr5290/src/main/java/org/talend/hadoop/distribution/emr5290/modulegroup/EMR5290HiveOnSparkModuleGroup.java [hunk 1]
```diff
@@ -23,6 +23,7 @@ public class EMR5290HiveOnSparkModuleGroup {
     public static Set<DistributionModuleGroup> getModuleGroups() {
         Set<DistributionModuleGroup> hs = new HashSet<>();
         hs.add(new DistributionModuleGroup(EMR5290Constant.SPARK_HIVE_MRREQUIRED_MODULE_GROUP.getModuleName(), true));
+        hs.add(new DistributionModuleGroup(EMR5290Constant.SPARK_HIVE_TEZ_MRREQUIRED_MODULE_GROUP.getModuleName(), true));
```
Comment (AlixMetivier) [inline main/plugins/org.talend.hadoop.distribution.emr5290/src/main/java/org/talend/hadoop/distribution/emr5290/modulegroup/EMR5290HiveOnSparkModuleGroup.java:26]: true here means that you will upload the libraries to HDFS, maybe you want to make it false by default ?

Comment (sanulich) [reply main/plugins/org.talend.hadoop.distribution.emr5290/src/main/java/org/talend/hadoop/distribution/emr5290/modulegroup/EMR5290HiveOnSparkModuleGroup.java:26]: no, I need this files in the HDFS.

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml [hunk 2]
```diff
@@ -59,6 +59,12 @@
         	id="azure-storage-5.0.0.jar"
         	name="azure-storage-5.0.0.jar"
         	mvn_uri="mvn:com.microsoft.azure/azure-storage/5.0.0"/>
+        	
+        <libraryNeeded
+        	context="plugin:org.talend.hadoop.distribution.emr5290"
+        	id="hadoop-lzo-0.4.19.jar"
+        	name="hadoop-lzo-0.4.19.jar"
+        	mvn_uri="mvn:org.talend.libraries/hadoop-lzo-0.4.19/6.0.0"/>
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:67]: Long time ago org.talend.libraries was used everywhere (I don't konw exactly why) but now we try to use official group.

Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:67]:
```suggestion
        	mvn_uri="mvn:com.hadoop.gplcompression/hadoop-lzo/0.4.19"/>
```

File: main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml [hunk 3]
```diff
@@ -2325,6 +2332,21 @@
                 <library id="orc-core-1.5.5-nohive.jar"/>
         </libraryNeededGroup>
         
+        <!-- Spark Hive Tez group -->
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2335]: This part of the fix is not in original TBD-13622 scope right ?

Comment (sanulich) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2335]: It's original scope. It's about second exception, that was write in the ticket comment.

Comment (lbourgeois) [reply main/plugins/org.talend.hadoop.distribution.emr5290/plugin.xml:2335]: Ok I thougth this issue was on Universal only not with the plain dedicated emr529 plugin

**Review by lbourgeois — CHANGES_REQUESTED** [review]
Small change on a maven group

**Review by lbourgeois — APPROVED** [review]
LGTM

**Pull request merged**